### PR TITLE
Move mx_EventTile_info out of mx_EventTile:not([data-layout=bubble])

### PR DESCRIPTION
**Pull request by luixxiul**
This PR moves `mx_EventTile_info` out of `mx_EventTile:not([data-layout=bubble])` to prevent an unexpected regression due to [`:not()` pseudo class which has a number of quirks, tricks, and unexpected results that you should be aware of before using it](https://developer.mozilla.org/en-US/docs/Web/CSS/:not#description).

This PR also aligns avatars and event texts on IRC layout.
Fixes https://github.com/vector-im/element-web/issues/22783

| |Before|After|
|-|---------|------|
|E2E icon (modern layout)|(unchanged)|after|
|Line (modern layout)|(unchanged)|after|
|E2E icon (IRC layout)|before1↑ Mind the block margin|after1|
|Line (IRC layout)|before1|after1|
| |before1|after1|

Signed-off-by: Suguru Hirahara <luixxiul@users.noreply.github.com>

type: defect
Notes: Align avatars and event texts on IRC layout

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

**Review by turt2live — DISMISSED** [review]
can we get screenshots of what the regression was and how it was fixed? Screenshots of the CSS don't really tell the story for what's going on here.

**Comment by luixxiul** [comment]
Screenshots were added.

File: res/css/views/rooms/_EventTile.scss [hunk 1]
```diff
@@ -247,6 +257,14 @@ $left-gutter: 64px;
         .mx_ReplyTile .mx_EventTileBubble {
             left: unset; // Cancel the value specified above for the tile inside ReplyTile
         }
+
+        &.mx_EventTile_info {
```
Comment (luixxiul) [inline res/css/views/rooms/_EventTile.scss:261]: For IRC layout

File: res/css/views/rooms/_EventTile.scss [hunk 2]
```diff
@@ -318,8 +336,30 @@ $left-gutter: 64px;
             padding-top: 0px !important;
         }
 
-        &.mx_EventTile_info .mx_EventTile_line {
-            padding-left: calc($left-gutter + 20px); // override padding-left $left-gutter
+        &.mx_EventTile_info {
```
Comment (luixxiul) [inline res/css/views/rooms/_EventTile.scss:339]: For modern layout

File: res/css/views/rooms/_EventTile.scss [hunk 3]
```diff
@@ -350,33 +390,6 @@ $left-gutter: 64px;
     font-size: $font-14px;
     position: relative;
 
-    &.mx_EventTile_info {
```
Comment (luixxiul) [inline res/css/views/rooms/_EventTile.scss:353]: For both IRC and modern layouts

File: res/css/views/rooms/_EventTile.scss [hunk 4]
```diff
@@ -424,10 +437,6 @@ $left-gutter: 64px;
     }
 
     &[data-layout=irc] {
-        .mx_EventTile_info .mx_EventTile_line {
-            padding-left: 0; // Override .mx_EventTile:not([data-layout="bubble"]).mx_EventTile_info .mx_EventTile_line
```
Comment (luixxiul) [inline res/css/views/rooms/_EventTile.scss:428]: Because overriding `:not()` pseudo class is no longer necessary.

**Review by robintown — APPROVED** [review]
LGTM

**Comment by robintown** [comment]
except it looks like you have merge conflicts

**Pull request merged**